src/codeu/chat/common/ConversationHeader.java

```diff
@@ -14,10 +14,12 @@
 
 package codeu.chat.common;
 
+import java.awt.List;
```

Comment:
This is probably not the List you wanted :p (this is a GUI list, not a data structure list)